src/views/partials/__footer.njk

```diff
@@ -5,28 +5,25 @@
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/privacy-notice" data-track-options="{&quot;dimension29&quot;:&quot;Privacy&quot;}" href="/help/privacy-notice">Privacy</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/cookies" data-track-options="{&quot;dimension29&quot;:&quot;Cookies&quot;}" href="/cookies">Cookies</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/cookies" data-track-options="{&quot;dimension29&quot;:&quot;Cookies&quot;}" href="/help/cookies">Cookies</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/contact" data-track-options="{&quot;dimension29&quot;:&quot;Contact&quot;}" href="https://www.gov.uk/government/organisations/companies-house#org-contacts">Contact Us</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/accessibility-statement" data-track-options="{&quot;dimension29&quot;:&quot;Accessibility statement&quot;}" href="/help/accessibility-statement">Accessibility statement</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/accessibility" data-track-options="{&quot;dimension29&quot;:&quot;Accessibility statement&quot;}" href="/accessibility-statement">Accessibility statement</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/contact" data-track-options="{&quot;dimension29&quot;:&quot;Contact&quot;}" href="/contact">Contact</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/developers" data-track-options="{&quot;dimension29&quot;:&quot;Developers&quot;}" href="/developers">Developers</a>
           </li>
           <li class="govuk-footer__inline-list-item">
-            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/help/terms-conditions" data-track-options="{&quot;dimension29&quot;:&quot;Terms and conditions&quot;}" href="/help/terms-conditions">Terms and conditions</a>
+            <a class="govuk-footer__link" data-track-category="footerClicked" data-track-action="supportLink" data-track-label="/prototype-kit" data-track-options="{&quot;dimension29&quot;:&quot;Prototype kit&quot;}" href="https://govuk-prototype-kit.herokuapp.com/">GOV.UK Prototype Kit {{releaseVersion}}</a>
```

Comment:
We don't need the gov uk prototype text, that's just on the prototype

Reply:
Oh okay thanks